### ISSUE TITLE
Incorrect imports in plugin-nested-docs readme

### ISSUE THREAD

**Issue by ChristianMay21** [payloadcms/payload#4365]
# Documentation Issue

In the plugin-nested-docs readme, there's some sample code with the following imports:
```
import createParentField from "@payloadcms/plugin-nested-docs/fields/parent";
import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields/breadcrumbs";
````

Using the most recent version of that plugin, it seems that createParentField and createBreadcrumbsField can be exported either from `@payloadcms/plugin-nested-docs/fields` or from `@payloadcms/plugin-nested-docs/dist/fields/parent` and `@payloadcms/plugin-nested-docs/dist/fields/breadcrumbs`, respectively.

It's possible I'm misunderstanding something, but if I am not, I believe those code samples may need to be updated.

**Comment by ChristianMay21** [comment]
@JessChowdhury are you sure it would include a `src` directory? Looking at `node_modules/@payloadcms/plugin-nested-docs/` on my project, there does not appear to be a `src` directory.

![image](https://github.com/payloadcms/payload/assets/45889402/fa675685-0686-4103-ad30-c21bbc69c32a)

**Comment by JessChowdhury** [comment]
@ChristianMay21 apologies I misunderstood the context here - you're correct that `createParentField` and `createBreadcrumbsField` can be exported from either @payloadcms/plugin-nested-docs/fields or directly from `@payloadcms/plugin-nested-docs/dist/fields/parent` and `@payloadcms/plugin-nested-docs/dist/fields/breadcrumbs`

I'll update the readme to use `@payloadcms/plugin-nested-docs/fields' as it is simpler.

**Comment by ChristianMay21** [comment]
Hi @JessChowdhury - no worries! I also thought that `@payloadcms/plugin-nested-docs/fields` was simpler, but when I tried to import it from there, I got a 'missing call signature' error from TypeScript when I tried to invoke the exported functions. For whatever reason, importing from the dist folder worked correctly.

It's possible that that was some sort of issue on my end, but my TypeScript environment is fairly standard, so I wanted to note it in case it was a general issue.

**Comment by jacobsfletch** [comment]
Hey @ChristianMay21 there was in fact an issue in the way we were exporting these field utilities. I've got a fix this queued up in #4462 (details there). This will allow you to use named imports as expected, like this:

```ts
import { createParentField } from "@payloadcms/plugin-nested-docs/fields";
```

**Comment by ChristianMay21** [comment]
Great - thanks for looking into it, and thanks for the fix @jacobsfletch !
